functions/New-DbaAvailabilityGroup.ps1

```diff
@@ -636,14 +636,14 @@ function New-DbaAvailabilityGroup {
                     }
                 }
                 if ($Database) {
-                    $null = Add-DbaAgDatabase -SqlInstance $Primary -SqlCredential $PrimarySqlCredential -AvailabilityGroup $Name -Database $Database -SeedingMode $SeedingMode -SharedPath $SharedPath -Secondary $Secondary -SecondarySqlCredential $SecondarySqlCredential
+                    $null = Add-DbaAgDatabase -SqlInstance $Primary -SqlCredential $PrimarySqlCredential -AvailabilityGroup $Name -Database $Database -SeedingMode $SeedingMode -SharedPath $SharedPath -UseLastBackup:$UseLastBackup -Secondary $Secondary -SecondarySqlCredential $SecondarySqlCredential
```

Comment:
-UseLastBackup $UserLastBackup is sufficient syntax, to match the rest of the call.

Reply:
Ah, nope never mind...forgot that was a switch parameter